x-pack/plugins/session_view/public/components/process_tree_alerts_filter/index.tsx

```diff
@@ -124,9 +124,10 @@ export const ProcessTreeAlertsFilter = ({
             {totalAlertsCount === filteredAlertsCount && (
               <FormattedMessage
                 id="xpack.sessionView.alertTotalCountStatusLabel"
-                defaultMessage="Showing {count} alerts"
+                defaultMessage="{count, plural, one {Showing <bold>#</bold> alert} other {Showing <bold>#</bold> alerts}}"
                 values={{
-                  count: <strong>{totalAlertsCount}</strong>,
+                  count: totalAlertsCount,
+                  bold: (str: string) => <strong>{str}</strong>,
```

Comment:
before and after image for that could be nice

Reply:
I don't have a quick way to have session view data in my local env, but check the unit test, it should cover it, at least the text is there. The markup should work as well, taken straight from the docs https://formatjs.io/docs/react-intl/components#rich-text-formatting. But if we have a quick way to have the session view in local, happy to test it manually